github-dark.css

```diff
@@ -5250,3 +5250,94 @@
   }
   /* Literal.Number.Integer.Long */
 }
+
+@-moz-document domain("github.com") {
+  /* ui when there are no workflows setup */
+  .deployments-timeline-item + div:not(.deployments-timeline-item) {
+    background-image: linear-gradient( rgba(28, 28, 28, 0), rgba(28, 28, 28, 1)) !important;
+  }
+  [style*="background-color: #e1e4e8"] {
+    background-color: #8e8e8e !important;
+  }
+  [style*="color: #e1e4e8"] {
+    color: #8e8e8e !important;
+  }
+  /**
+   * this is the same form as when you're committing from the site
+   * this means there's a small arrow on the left (which points at
+   * the user's avatar) and this ends up showing in the dropdown too
+   */
+  .file-commit-form .dropdown-menu .commit-form:after {
+    content: none !important;
+  }
+  .file-commit-form .commit-form {
+    background: #181818 !important;
+  }
+}
+
+@-moz-document domain("launch-editor.github.com") {
+  body {
+    background: #181818 !important;
+  }
+  .editor-node[style*="background-color: rgb(68, 77, 85)"] {
+    background: #343434 !important;
+  }
+  .editor-node {
+    background: #1c1c1c !important;
+  }
+  .blankslate-box {
+    background: #242424 !important;
+  }
+  .overlay {
+    background: rgba(27, 31, 35, .5) !important;
+  }
+  .blankslate-box, .editor-node, .editor-node > div {
+    border-color: #484848 !important;
+  }
+  .input, .output {
+    background-image: none !important;
+    background-color: #484848 !important;
+  }
+  .input:before, .output:before {
+    background: transparent !important;
+  }
+  .circle.live:after {
+    background-color: /*[[base-color]]*/ !important;
```

Comment:
Need to the fallback color to allow people to install `github-dark.css` to all these token comments. See existing styles on how to do them. Basically, you add the color after the comment in lines like this. On things like `box-shadow` you need to add a second rule.